src/Features/Context/TiqrContext.php

```diff
@@ -40,65 +46,48 @@
  *
  * @SuppressWarnings(PHPMD)
  */
-class TiqrContext implements Context, KernelAwareContext
+class TiqrContext implements Context
 {
-    /**
-     * @var MinkContext
-     */
-    protected $minkContext;
-
-    /**
-     * @var KernelInterface
-     */
-    protected $kernel;
+    protected MinkContext $minkContext;
 
-    protected $metadataUrl;
+    protected string $metadataUrl;
 
-    protected $clientSecret;
+    protected string $clientSecret;
 
-    protected $userAgent;
+    protected string $userAgent;
 
     /**
      * The registration metadata.
-     * @var mixed
      */
-    protected $metadata;
+    protected stdClass $metadata;
```

Comment:
PHP is moving away from dynamic properties, so maybe we have an opportunity here to replace de stdClass with  an actual object?

Reply:
We certainly could. But this is not a thing I'm comfortable doing in this upgrade. One for the backlog.